libs/net/dal/Models/ContentFileReference.cs

```diff
@@ -140,6 +140,7 @@ public ContentFileReference(Content content, IFormFile file)
         this.UpdatedBy = content.UpdatedBy;
         this.UpdatedById = content.UpdatedById;
         this.UpdatedOn = DateTime.UtcNow;
+        this.SourceFile = "";
```

Comment:
This issue was introduced in the Video/Audio editor PR. I'm not certain the best way to resolve it. For now I resolved the linting issue.